Fix test array property py3

rodrigc opened the pull request:
## Contributor Checklist:

* [x] There is an associated issue.
      [Create a new one](https://github.com/LeastAuthority/txkube/issues/new) if necessary.
      Link to it below.
* [x] I have created a [newsfragment](http://towncrier.readthedocs.io/en/actual-freaking-docs/quickstart.html#creating-news-fragments) in src/txkube/newsfragments/
* [x] I have updated the automated tests.

##

Fixes #188

rodrigc committed:
Add newsfragment

exarkun commented on src/txkube/test/test_swagger.py:
```diff
@@ -584,6 +590,32 @@ def test_array_property(self):
+                expected_types=(unicode,),
+                actual_type=int,
+            ),
+        )
```
Instead of doubling this test, how about replacing the bad value with a less fraught type?  For example, `add=[0]` should fail roughly the same way as `b"hello"` does but without the varying behavior between Python 2 and Python 3.

rodrigc replied:
Sounds good to me.  I have updated the test to use `add=[0]`.

rodrigc committed:
Use an int value in test_array_property.

This eliminates differences in the test between Python 2 and 3
due to unicode/bytes/native string issues.
